src/pages/Home/Home.page.jsx

```diff
@@ -1,16 +1,23 @@
 import React, { useContext } from 'react';
-import HomeVideoGrid from '../../components/HomeVideoGrid/HomeVideoGrid.component';
-import { StoreContext } from '../../utils/store/store-context';
-
+import { StoreContext } from '../../utils/store/StoreContext';
+import VideoGrid from '../../components/VideoGrid/VideoGrid.component';
 import classes from './Home.module.css';
 
 const HomePage = () => {
-  const { searchParamTitle } = useContext(StoreContext);
+  const { store } = useContext(StoreContext);
+  const { searchParamTitle, videoList, fetchIsLoading, fetchError } = store;
 
   return (
     <section className={classes['home-container']}>
       <h1>{searchParamTitle}</h1>
-      <HomeVideoGrid />
+      {!fetchIsLoading && videoList.length > 0 && (
+        <VideoGrid videoList={videoList} />
+      )}
+      {!fetchIsLoading && videoList.length === 0 && !fetchError && (
+        <p>Content not found</p>
+      )}
+      {!fetchIsLoading && fetchError && <p>{fetchError}</p>}
```

Comment:
I think if you leave just `fetchError` is ok, because if you get an error at some point in the fetch and the loading is never set to false you still want to show the error